manual/commands.md

```diff
@@ -0,0 +1,633 @@
+# Help
+Once installed, if you run the `meza` command all by itself, it will display *help* like the following:
+```bash
+Mediawiki EZ Admin
+
+Usage: meza COMMAND [directives]
+
+To setup a multi-server environment, do:
+
+$ meza setup env # Setup the environment, following prompts
+# Edit config as required:
+$ sudo vi /opt/conf-meza/secret/<env-name>/hosts
```

Comment:
Does the setup command above create these files? Or are they part of the installation?

Reply:
Can you clarify which things are required and which are optional?